Bump abstract to 0.22.1

Buckram123 opened the pull request:
### Checklist

- [x] CI is green.
- [x] Bump abstract packages versions
- [x] Bump abstract contracts versions
- [x] Bump modules versions
- [x] Migration clone tests passing
- [x] Update changelog

Kayanski reviewed (commented):
Are we sure that the migration of version control works. Because we have changed config for this module. Have you thought about that ?

Buckram123 commented:
> Are we sure that the migration of version control works. Because we have changed config for this module. Have you thought about that ?

Added a test to cover that

Kayanski reviewed (approved):
LGTM

Kayanski commented on interchain/scripts/README.md:
```diff
@@ -0,0 +1,55 @@
+
+The first step in deploying abstract is deploying Abstract on the 2 chains you want to connect.
+
+### Deploy and Connect Polytone
+
+Then you need to deploy and connect Polytone between the 2 chains. You should find a complete tutorial inside the [abstract-polytone repo](https://github.com/AbstractSDK/polytone/blob/abstract/deployment/DEPLOY.md).
+
+This comes in 3 steps:
+
+#### Deploy the Polytone contracts on each chain
+
+Upload the polytone contracts (voice, note and proxy at least) on each chain. A script is available in the [abstract-polytone repo](https://github.com/AbstractSDK/polytone/blob/abstract/deployment/scripts/src/bin/upload_contracts.rs).
+
+#### Instantiate the contracts on each chain
+
+In order to create a one-directional connection, you need to instantiate the note on the local side and the voice on the remote side. A script is available in the [abstract-polytone repo](https://github.com/AbstractSDK/polytone/blob/abstract/deployment/scripts/src/bin/instantiate_chains.rs).
```
The deploy-ibc script deploys abstract-IBC on one chain. Now this is integrated in the deploy script, so this is not needed anymore
As per the second one, sure, why not :)

adairrr replied:
Asking because it can reduce the number of steps significantly. In #333 it also ensures that when Abstract is deployed then the IBC infra is deployed. IN writing this, however, it doesn't guartantee that a loaded abstract instance has the IBC infra present, which could lead to inconsistencies.
